ckanext/harvest/logic/action/get.py

```diff
@@ -310,6 +312,38 @@ def harvesters_info_show(context,data_dict):
 
     return available_harvesters
 
+@side_effect_free
+def harvest_log_list(context,data_dict):
+    '''Returns a list of harvester log entries grouped by level.
```

Comment:
AFAICT the output is a dict with keys being the log levels and values lists of messages